<?xml version="1.0"?>
<launch>
    <node pkg="move_base" type="move_base" respawn="true" name="move_base" output="screen" >

        <rosparam file="$(find neo_bot)/params/local_costmap_params.yaml" command="load" />
        <rosparam file="$(find neo_bot)/params/global_costmap_params.yaml" command="load" />
        <rosparam file="$(find neo_bot)/params/base_local_planner_params.yaml" command="load" /> 
        <rosparam file="$(find neo_bot)/params/move_base_params.yaml" command="load" />
        <rosparam file="$(find neo_bot)/params/global_planner_params.yaml" command="load"/>

        <param name="base_global_planner" value="global_planner/GlobalPlanner" />
        <param name="base_local_planner" value="base_local_planner/TrajectoryPlannerROS"/> 

    </node> 
</launch>
